.gitignore

```diff
@@ -59,3 +59,7 @@ typings/
 
 # next.js build output
 .next
+
+application.css
+.scss-cache
```

Comment:
I think it's `.sass-cache` what you should add here. I think the folder explained in the trello card is not correct (unless 2 folders were generated for some reason)